Cargo.toml

```diff
@@ -11,11 +12,14 @@ repository = "https://github.com/keep-starknet-strange/zaun/"
 version = "0.1.0"
 
 [workspace.dependencies]
-ethers = "2.0.7"
+ethers = { git = "https://github.com/gakonst/ethers-rs", rev = "f0e5b194f09c533feb10d1a686ddb9e5946ec107" }
```

Comment:
why not stable ?

Reply:
Coz anvil doesn't: https://github.com/foundry-rs/foundry/blob/master/Cargo.toml#L191
And it causes mismatches in the way calls are encoded/decoded, making it unusable